src/globals.js

```diff
@@ -0,0 +1,9 @@
+import global from "global";
+
+export const window = global.window;
+export const console = global.console;
+export const document = global.document;
```

Comment:
I think this should be `window && window.document`, since `window` is not necessarily `global`. Likewise for `navigator` and probably `sessionStorage`.